as/itests/org.jboss.tools.as.ui.bot.itests/src/org/jboss/tools/as/ui/bot/itests/parametized/server/ShowInContextMenuTest.java

```diff
@@ -0,0 +1,87 @@
+ /*******************************************************************************
+ * Copyright (c) 2007-2021 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v 1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributor:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.as.ui.bot.itests.parametized.server;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import org.eclipse.reddeer.eclipse.wst.server.ui.cnf.Server;
+import org.eclipse.reddeer.eclipse.wst.server.ui.cnf.ServersView2;
+import org.eclipse.reddeer.junit.runner.RedDeerSuite;
+import org.eclipse.reddeer.requirements.server.ServerRequirementState;
+import org.eclipse.reddeer.swt.api.Browser;
+import org.eclipse.reddeer.swt.api.Menu;
+import org.eclipse.reddeer.swt.impl.browser.InternalBrowser;
+import org.eclipse.reddeer.swt.impl.menu.ContextMenu;
+import org.jboss.ide.eclipse.as.reddeer.server.requirement.ServerRequirement.JBossServer;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * Test class for testing options in Show In context menu
+ * 
+ * @author Lukas Grossmann
+ * 
+ * TODO: Add more tests for Show In context menu options
+ */
+@RunWith(RedDeerSuite.class)
+@JBossServer(state=ServerRequirementState.PRESENT)
+public class ShowInContextMenuTest {
+	
+	private final ServersView2 sv = new ServersView2();
+	private final Menu menu = new ContextMenu();
+	private final Server server = sv.getServer("WildFly 24+ Server");
```

Comment:
Remove final keyword from all the class members, please. Instances of such objects could change in time and so it is much better approach to reinitialize them in Before method and clear them in after method so that the test env. is always the same and prepared for all test cases/methods.